.github/workflows/lint.yml

```
@@ -0,0 +1,16 @@
+name: Lint userstyles
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    name: Lint
```

Matching case with the other workflows